bug_view_limited_inc.php

```diff
@@ -0,0 +1,676 @@
+<?php
```

Comment:
I have done a fair bit of work over time to reduce the number of pages we have.  For example:

- Combine simple/advanced pages.
- Combine view/print pages.

I really would rather avoid having yet another page to view the issue.  As I said in other comments, if we can drive what should appear via the access level (VIEWER), that would make a lot of sense.  If the developer team doesn't want something to show, then avoid sharing the url or use private notes for example.  There is likely gaps between what this approach can do and the requirements you are trying to achieve that may be worth a discussion.

I also expect what you think should be viewed in public/private issues when viewed via a share url may be slightly different from what others believe should appear.  Another thing that we need to think about.